### DISCUSSION TITLE
"Recherche & Stats" : ajouter variateur du nombre de lignes affichées à la pagination

### DISCUSSION THREAD

**Discussion by Aurely9n** [JeromeDevome/GRR#309] [Idées]
Bonjour,

Je propose que soit ajouté sur la page de "Recherche et Stats" (app.php?p=report) un variateur du nombre de ligne à afficher (réponses à retourner) lorsque les réponses sont affichées.
Pour le moment, seulement 10 réponses sont affichées et il faut naviguer avec la pagination pour retrouver voir ses infos.

Il serait bien d'ajouter un champ supplémentaire à coté de la pagination permettant de modifier le nombre de réponses avec des pas allant de "10", "20", "50", "100" et "Tous" par exemple.

Bien à vous

**Comment by ynaessens** [comment]
Bonjour,
dans la version "old look" 3.5.1, c'est par défaut "tous" ;-)
Cordialement,
YN

**Reply by Aurely9n** [reply]
Oui pardon, j'aurai du précisé que j'utilise actuellement la version 4.2.0 de l'app' et là ben c'est 10 par défaut. Ou alors je n'ai pas vu où pouvais se configurer ce paramètre dans l'administration.

Cordialement,
Aurélien
